pkg/schedule/checker/replica_strategy.go

```diff
@@ -123,8 +128,12 @@ func (s *ReplicaStrategy) swapStoreToFirst(stores []*core.StoreInfo, id uint64)
 // SelectStoreToRemove returns the best option to remove from the region.
 func (s *ReplicaStrategy) SelectStoreToRemove(coLocationStores []*core.StoreInfo) uint64 {
 	isolationComparer := filter.IsolationComparer(s.locationLabels, coLocationStores)
+	level := constant.High
```

Comment:
The default level of some operators is medium.

Reply:
yes, I have search the call place, all of them are medium